src/Tkmm/App.axaml.cs

```diff
@@ -199,17 +204,48 @@ public static void Focus()
     public static void Toast(string message, string? title = null, NotificationType type = NotificationType.Information,
         TimeSpan? expiration = null, Action? action = null)
     {
-        Dispatcher.UIThread.Invoke(() => {
-            _notificationManager?.Show(
-                new Notification(title ??= Locale[TkLocale.NoticePopupMessage], message, type, expiration, action));
-        });
+        try {
+            Dispatcher.UIThread.Invoke(() => {
+                if (_notificationManager != null) {
+                    string safeTitle = title;
+                    try {
+                        if (safeTitle == null) {
+                            safeTitle = Locale[TkLocale.NoticePopupMessage];
+                        }
+                    }
+                    catch {
+                        safeTitle = "Notice";
+                    }
+                    
+                    _notificationManager.Show(
+                        new Notification(safeTitle, message, type, expiration, action));
+                }
```

Comment:
Message defaulting is already handled by the LocaleManager. If this entry is missing, a compile-time error will be raised.